src/test/java/api/InstancesApiExamples.java

```diff
@@ -460,37 +444,45 @@ public void canUpdateAnExistingInstance()
   }
 
   @Test
-  public void canUpdateAnExistingInstanceWithPreceedingSucceedingTitlesMarcSource()
-    throws InterruptedException,
-    MalformedURLException,
-    TimeoutException,
-    ExecutionException {
-
+  public void canUpdateAnExistingInstanceWithPrecedingSucceedingTitlesMarcSource() {
     UUID id = UUID.randomUUID();
 
     JsonObject smallAngryPlanet = smallAngryPlanet(id);
     smallAngryPlanet.put("natureOfContentTermIds",
       new JsonArray().add(ApiTestSuite.getBibliographyNatureOfContentTermId()));
     smallAngryPlanet.put(PUBLICATION_PERIOD_KEY, publicationPeriodToJson(new PublicationPeriod(1000, 2000)));
 
-    JsonArray procedingTitles = new JsonArray();
-    procedingTitles.add(
+    JsonArray precedingTitles = new JsonArray();
+    precedingTitles.add(
       new JsonObject()
         .put("title", "Chilton's automotive industries")
         .put("identifiers", new JsonArray().add(
           new JsonObject()
             .put("identifierTypeId", "913300b2-03ed-469a-8179-c1092c991227")
             .put("value", "0273-656X"))
         ));
-    smallAngryPlanet.put(PRECEDING_TITLES_KEY, procedingTitles);
+    smallAngryPlanet.put(PRECEDING_TITLES_KEY, precedingTitles);
     smallAngryPlanet.put("source", "MARC");
 
     JsonObject newInstance = createInstance(smallAngryPlanet);
 
+    precedingTitles = new JsonArray();
+    precedingTitles.add(
+      new JsonObject()
+        .put("title", "Chilton's automotive industries")
+        .put("id", newInstance.getJsonArray("precedingTitles").getJsonObject(0).getString( "id" ))
+        .put(PrecedingSucceedingTitle.SUCCEEDING_INSTANCE_ID_KEY, smallAngryPlanet.getString("id"))
+        .put("identifiers", new JsonArray().add(
+          new JsonObject()
+            .put("identifierTypeId", "913300b2-03ed-469a-8179-c1092c991227")
+            .put("value", "0273-656X"))
+        ));
+
     JsonObject updateInstanceRequest = newInstance.copy()
       .put(TAGS_KEY, new JsonObject().put(TAG_LIST_KEY, new JsonArray().add(tagNameTwo)))
       .put(PUBLICATION_PERIOD_KEY, publicationPeriodToJson(new PublicationPeriod(2000, 2012)))
-      .put(PRECEDING_TITLES_KEY, procedingTitles)
+      // Deliberately send update request with preceding title without an ID
+      .put(PRECEDING_TITLES_KEY, precedingTitles)
```

Comment:
The code on line 473 above, seems to deliberately include the `id` property in this representation.

That does not fit with this comment which states that it is deliberately not including the `id`.

Am I missing something in how these line up together?

(the tags UI currently sends the full representation with an amendment to the tags. As I understand it, the situation where the ID is not present is the general edit, which is being changed concurrently)

Reply:
@marcjohnson-kint Yes, you are right. I removed this comment from the codebase. Thank you!